docs/copied-from-beats/shared-path-config.asciidoc

```diff
@@ -86,6 +86,9 @@ Example:
 path.data: /var/lib/beats
 ------------------------------------------------------------------------------
 
+TIP: When running multiple Beat instances on the same host, make sure they
```

Comment:
can we either use the beatname or remove `Beat` altogether from this sentence?

Reply:
updated 587c1a8e